packages/stress/src/utils/stressClient.ts

```diff
@@ -235,12 +254,22 @@ export class StressClient {
             )
             logger.log('joinSpace transaction', issued)
         }
-        await this.startStreamsClient({ spaceId })
+        await this.startStreamsClient({ metadata: { spaceId } })
         await this.streamsClient.joinStream(spaceId)
         await this.streamsClient.joinStream(makeDefaultChannelStreamId(spaceId))
     }
 
     async stop() {
+        await this.exportDevice()
         await this.streamsClient.stop()
     }
+
+    async exportDevice(): Promise<ExportedDevice | undefined> {
+        const device = await this.streamsClient.cryptoBackend?.encryptionDevice.exportDevice()
+        if (device) {
+            await fs.writeFile(this.deviceFilePath, JSON.stringify(device, null, 2))
```

Comment:
can you try catch around the write file?

Reply:
nice catch! *ba dum tss